gmaildump/gmailhistory.py

```diff
@@ -233,17 +269,20 @@ def store_msgs_in_db(self, msgs_list):
 
     def get_default_ts(self):
         '''
-        Gives next day date from current day
-        :rtype: str(returns tommorrow date)
+        This fun helps to return next day date from today in Y/m/d format
```

Comment:
It is not necessary to always tell 'this fun helps....', concise docstring is good, and more over, i think, if you even give proper naming to you functions, that will be also not required. So, In case you find that your function name describes its work pretty well, the docstring will only contain, params its expecting and return type. Please follow if it makes sense, in every function.